### PR TITLE
Swift imple of #1665, #1674.

### PR DESCRIPTION
**Pull request by hanjoes**

### CODE REVIEW

File: runtime/Swift/Sources/Antlr4/Parser.swift [hunk 1]
```diff
@@ -600,6 +593,24 @@ open class Parser: Recognizer<ParserATNSimulator> {
         }
         return o
     }
+    
+    /** How to create a token leaf node associated with a parent.
+     *  Typically, the terminal node to create is not a function of the parent.
+     *
+     *  @since 4.6.1
```
Comment (parrt) [inline runtime/Swift/Sources/Antlr4/Parser.swift]: @hanjoes can you do a global check for 4.6.1? should be 4.7. thanks!

Comment (hanjoes) [inline runtime/Swift/Sources/Antlr4/Parser.swift]: oh sure, will do it tonight.

Comment (parrt) [inline runtime/Swift/Sources/Antlr4/Parser.swift]: actually, i'm merge then tweak!

Comment (hanjoes) [inline runtime/Swift/Sources/Antlr4/Parser.swift]: sounds good.

**Pull request merged**